src/shared/components/settings/SettingsMenu.tsx

```diff
@@ -0,0 +1,433 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import autobind from 'autobind-decorator';
+import {
+    buildDriverAnnotationControlsHandlers,
+    buildDriverAnnotationControlsState,
+    IDriverAnnotationControlsHandlers,
+    IDriverAnnotationControlsState,
+    IDriverSettingsProps,
+    IExclusionSettings,
+} from '../../driverAnnotation/DriverAnnotationSettings';
+import InfoIcon from '../InfoIcon';
+import styles from './styles.module.scss';
+import classNames from 'classnames';
+import DriverAnnotationControls from 'shared/components/driverAnnotations/DriverAnnotationControls';
+import { computed, observable } from 'mobx';
+
+enum EVENT_KEY {
+    showPutativeDrivers = '0',
+    showPutativePassengers = '1',
+    showUnknownOncogenicity = '2',
+    showGermlineMutations = '3',
+    showSomaticMutations = '4',
+    showUnknownStatusMutations = '5',
+    showUnknownTier = '6',
+    hideUnprofiledSamples = '7',
+    toggleAllMutationStatus = '8',
+    toggleAllDriverAnnotation = '9',
+    toggleAllDriverTiers = '10',
+}
+
+function boldedTabList(tabs: string[]) {
+    return (
+        <span>
+            {tabs.map((tab, index) => (
+                <span>
+                    <strong>{tab}</strong>
+                    {index < tabs.length - 1 ? ', ' : ''}
+                </span>
+            ))}
+        </span>
+    );
+}
+
+export interface IAlterationFilterSettings {
+    store: IDriverSettingsProps & IExclusionSettings;
+    infoElement?: JSX.Element | undefined;
+    disabled?: boolean;
+    customDriverSourceName?: string;
+    showDriverAnnotationSection?: boolean;
+    showTierAnnotationSection?: boolean;
+}
+
+@observer
+export default class SettingsMenu extends React.Component<
+    IAlterationFilterSettings,
+    {}
+> {
+    private driverSettingsState: IDriverAnnotationControlsState;
+    private driverSettingsHandlers: IDriverAnnotationControlsHandlers;
+    @observable _driverAnnotationsCheckboxToggle = true;
+    @observable _driverTiersCheckboxToggle = false;
+
+    constructor(props: IAlterationFilterSettings) {
+        super(props);
```

Comment:
you'll need to call `makeObservable` here

Reply:
we should run @adamabeshouse script to detect missing makeObservable